Dockerfile

```diff
@@ -49,6 +49,15 @@ RUN --mount=type=cache,target=/var/cache/apt,sharing=locked \
       /home/${USERNAME}/.ros && \
     chown -R $USER_UID:$USER_GID /home/${USERNAME} /opt/overlay_ws/
 
+# IMPORTANT: Optionally install Nvidia drivers for improved simulator performance with Nvidia GPUs.
+# To do this you must
+# 1. Uncomment the ENV and RUN entries below
+# 2. Replace the 'nvidia-driver-555' apt package with the Nvidia driver version on your host, e.g. nvidia-driver-535, nvidia-driver-555. Use nvidia-smi on your host to determine the driver version.
+# After rebuilding via `moveit_pro build` verify the drivers are active in your container by running `nvidia_smi` inside of `moveit_pro shell`.
+# ENV DEBIAN_FRONTEND=noninteractive
+# RUN apt update && apt install -y software-properties-common
```

Comment:
Aren't these two things already done?

Reply:
Hmmm...I do see them in the moveit_studio/Dockerfile (isn't that where `FROM ${MOVEIT_STUDIO_BASE_IMAGE} AS base` comes from)?

BUT, yesterday I removed the `ENV DEBIAN_FRONTEND=noninteractive` to see what would happen and it seemed to be waiting for user input while installing the drivers about keyboard language.

Reply:
> ENV DEBIAN_FRONTEND=noninteractive
Chat GPT is telling me ARGs are not inherited from the base image, which would be consistent with my experience.

> RUN apt update && apt install -y software-properties-common
I think software-properties-common should be installed by the base image. But I think verifying it here should be really fast so I'd prefer to keep it in case someone needs to copy paste this hack into a diff Dockerfile where that might not be true.

Reply:
> Chat GPT is telling me ARGs are not inherited from the base image, which would be consistent with my experience.

yeah but it's an ENV, not an ARG. ChatGPT is lying to you as usual ;)

but if you want to leave it here for copy pasta, ok